### ISSUE TITLE
Unable to resolve dependency

### ISSUE THREAD

**Issue by IgnacioRodriguez98** [Apollo3zehn/PureHDF#31]
Hi there, I made a LSTM with keras.net.

But since I couldn't set the seed, I decided to use tensroflow.net, the problem is that when I tried to install tensorlfow.keras show me the next error.

_**Unable to resolve dependency 'PureHDF'. Source(s) used: 'nuget.org', 'Microsoft Visual Studio Offline Packages'.**_

Anybody have Idea about how to solve this issue?

**Comment by Apollo3zehn** [comment]
Hi, thanks for you question. I am aware of that issue from some other Stackoverflow question. What kind of environment do you use?

- .NET Version
- IDE + Version (Visual Studio?, VS Code?)
- Nuget dependency style: `packages.config` file vs `<PackageReference>...</PackageReference>`

Thank you :-)

I suspect that on some environments it is a problem, that PureHDF is an alpha release.
